pkg/cmd/create/vault/create_vault.go

```diff
@@ -312,3 +314,50 @@ func (o *CreateVaultOptions) exposeVault(vaultService string) error {
 	}
 	return upgradeIngOpts.Run()
 }
+
+// CreateAuthServiceAccount creates a Serivce Account for the Auth service for vault
+func CreateAuthServiceAccount(client kubernetes.Interface, vaultName, serviceAccountName string, namespace string) (string, error) {
```

Comment:
I think this might be a rebase issue. This and the following three methods are obsolete and unused.